daemon.go

```diff
@@ -95,6 +95,16 @@ func (d *Daemon) DialNodeConsole(label string, token *Token) (io.ReadCloser, err
 	return node.OBM.DialConsole()
 }
 
+func (d *Daemon) PowerOnNode(label string, token *Token) error {
+	d.Lock()
+	defer d.Unlock()
+	node, err := d.getNodeWithToken(label, token)
+	if err != nil {
+		return err
+	}
+	return node.OBM.PowerOn()
```

Comment:
I think it's probably time to factor this pattern out;  we've got the same copypasta across three methods now. Maybe something like:

```go
func (d *Daemon) PowerOnNode(label string, token *Token) error {
        d.usingNodeWithToken(label, token, func(n *Node) error {
                return n.OBM.PowerOn()
        }
}
```

with an appropriate definition of `usingNodeWithToken`.